libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -92,6 +93,10 @@ class AP_BattMonitor
     // healthy - returns true if monitor is functioning
     bool healthy(uint8_t instance) const;
     bool healthy() const { return healthy(AP_BATT_PRIMARY_INSTANCE); }
+    
+    // is_powering_off - returns true when power button commands power off
+    bool is_powering_off(uint8_t instance) const;
+    bool is_powering_off() const { return is_powering_off(AP_BATT_PRIMARY_INSTANCE); }
```

Comment:
This should simply return true if any of the instances are shutting off, rather then just the primary.